### PR TITLE
doc: reorganize docs, navigation, consolidate cli reference to one page

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: doc/md/cli.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+## atlas env
+
+Print atlas env params
+
+
+Env prints atlas environment information.
+Every set environment param will print in the form of NAME=VALUE.
+
+List of supported environment parameters:
```
Comment (yonidavidson) [inline doc/md/cli.md:24]: This doesn't look good in the rendered document, maybe add `\n`
image

**Review by yonidavidson — APPROVED** [review]
Please see my comment before merge

File: cmd/atlas/doc.tmpl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Introduction
+
+This document serves as reference documentation for all available commands in the Atlas CLI.
+Similar information can be obtained by running any atlas command with the `-h` or `--help`
+flags.
+
+For a more detailed introduction to the CLI capabilities, head over to the
+[Getting Started](/cli/getting-started/setting-up) page.
+
+{{ range .Blocks }}
```
Comment (a8m) [inline cmd/atlas/doc.tmpl:18]: You can use `-` to remove all the redundant new lines 
```suggestion
{{- range .Blocks }}
```

**Review by a8m — APPROVED** [review]

**Comment by a8m** [comment]
Let's keep all URLs lowercased.

**Review by yonidavidson — APPROVED** [review]

**Pull request merged**